### ISSUE TITLE
Homepage Needed

### ISSUE THREAD

**Issue by HittmanA** [Code-Adic-Team/Code-Adic-source-code#2]
Hello there! If someone could make me a homepage it would be greatly appreciated! Do whatever you think will look/behave the best!

**Comment by EdwardHamHam** [comment]
Well, what's the site going to be like exactly?

**Comment by HittmanA** [comment]
Its gonna be a site where people can write HTML Javascript CSS or PHP code and then share it with the world.

**Comment by EdwardHamHam** [comment]
Okay, I can make us a fully custom site, using HTML and JavaScript. I'll start right away.

**Comment by HittmanA** [comment]
Thanks! I will make you owner of the organization to!

**Comment by HittmanA** [comment]
@Edwardthedog2 Hows the homepage coming along? If you need somewhere to display it live send me the file and I will add it to my website.

**Comment by EdwardHamHam** [comment]
Oh it's coming along nicely. I'm at school, and I'm working on it on my home computer, so I can view it as I work. I'll be working on it some more when I get home. At my school every student gets a laptop, but there just Chromebooks. I can send you the file when I get home, because it's on my windows 10 computer at home.

**Comment by HittmanA** [comment]
Ok thanks. Glad your helping me 😀!

**Comment by HittmanA** [comment]
@Edwardthedog2 Could you let me see the current Code Adic homepage that you made?

**Comment by EdwardHamHam** [comment]
Yes, tomorrow morning, when I first turn on my computer. Ill send a zip file to you with all of the images and the html files in it. Just unzip it and double click the index.html file. At least you shold be able to open html files that way. Are you on Windows?

**Comment by HittmanA** [comment]
No Im on Mac, but don't worry I have plenty of experience testing and opening html files. Thanks again by the way!

**Comment by EdwardHamHam** [comment]
[Homepage.zip](https://github.com/Code-Adic-Team/Code-Adic-source-code/files/243643/Homepage.zip)
Here's what it looks like so far.

**Comment by HittmanA** [comment]
Thanks!

**Comment by EdwardHamHam** [comment]
You're welcome!

**Comment by HittmanA** [comment]
Let me know when you finish the homepage please.

**Comment by EdwardHamHam** [comment]
Okay, I will.

**Comment by HittmanA** [comment]
Is the homepage almost finished?

**Comment by EdwardHamHam** [comment]
Yes, almost. Sorry, it's not done by now, I've been busy with school stuff.

**Comment by HittmanA** [comment]
Ok thank. If you need to take some time off to do school stuff I understand. Could you check out the new 404 page real quick? [404](http://codeadic.ahitt-industries.com/bla)

**Comment by EdwardHamHam** [comment]
Lol, yes, I like it, but I think maybe we should add like a link to the index.html homepage, so people can go back to the main site.

**Comment by HittmanA** [comment]
I will once we have it.

**Comment by EdwardHamHam** [comment]
okay, cool

**Comment by HittmanA** [comment]
Also feel free to upload new code to the repo.

**Comment by EdwardHamHam** [comment]
I am so sorry, but I had to factory reset my computer so I lost all my work! Sorry.

**Comment by HittmanA** [comment]
It's ok. Do you think you could redo the work or is it to much?

**Comment by EdwardHamHam** [comment]
I can redo it, I'll just need some time. I'm on summer vacation now, so I can work faster this time.

**Comment by HittmanA** [comment]
Hey @Edwardthedog2 sorry I've been so in-active. Im back now! How is the homepage coming?

**Comment by EdwardHamHam** [comment]
Sorry for the late reply, @HittmanA but I'm afraid I can not continue this homepage. I'm sorry, but I have become too busy with other things, I'm afraid, and for that I am sorry.

**Comment by HittmanA** [comment]
Thank you for replying. I understand about being busy, let me know if you want to try it again in the future!

**Comment by EdwardHamHam** [comment]
Okay, I will.

**Comment by farazfazli** [comment]
I can do this @HittmanA @Edwardthedog2

**Comment by HittmanA** [comment]
Ok thanks @farazfazli it will be much appreciated!

**Comment by EdwardHamHam** [comment]
Lol. Believe it or not, I actually just realized that the zip file I sent you before, @HittmanA  was the wrong site! It was the code for another site that I had been making for a software I own. Back then I kept all of my code for websites I was working on, in the same folder on my desktop. I guess I zipped up the wrong subfolder by mistake. Anyway, I'm not nearly as busy now, and plus I have a much faster computer now. So I can try and help again some if you still need it. 😃

**Comment by HittmanA** [comment]
Hey!

**Comment by HittmanA** [comment]
@Edwardthedog2 what languages (coding) are you good with?

**Comment by EdwardHamHam** [comment]
I'm best with HTML.

**Comment by HittmanA** [comment]
How about JavaScript?

**Comment by EdwardHamHam** [comment]
Well, I am learning JavaScript,  but it's not my current strong suit. But I can still work in it.

**Comment by matthew4444** [comment]
i can do it

**Comment by matthew4444** [comment]
and tomorows my birthday

**Comment by HittmanA** [comment]
Happy belated birthday!
